packaging/examples/metrics/kube-state-metrics/configmap.yaml

```
@@ -135,3 +135,73 @@ data:
+            version: v1beta2
+            kind: KafkaConnect
+          metricNamePrefix: strimzi_kafka_connect
+          metrics:
+            - name: resource_info
+              help: "The current state of a Strimzi Kafka Connect resource."
+              each:
+                type: Info
+                info:
+                  labelsFromPath:
+                    name: [ metadata, name ]
+              labelsFromPath:
+                exported_namespace: [ metadata, namespace ]
+                deprecated: [ status, conditions, "[reason=DeprecatedFields]", type ]
+                ready: [ status, conditions, "[type=Ready]", status ]
+                generation: [ status, observedGeneration ]
+                connectorPluginsClass: [ status, connectorPlugins, class ]
+                connectorPluginsType: [ status, connectorPlugins, type ]
+                connectorPluginsVersion: [ status, connectorPlugins, version ]
+                replicas: [ status, replicas ]
+                labelSelector: [ status, labelSelector ]
+        - groupVersionKind:
+            group: kafka.strimzi.io
+            version: v1beta2
+            kind: KafkaConnector
+          metricNamePrefix: strimzi_kafka_connector
+          metrics:
+            - name: resource_info
+              help: "The current state of a Strimzi Kafka Connector resource."
+              each:
+                type: Info
+                info:
+                  labelsFromPath:
+                    name: [ metadata, name ]
+              labelsFromPath:
+                exported_namespace: [ metadata, namespace ]
+                deprecated: [ status, conditions, "[reason=DeprecatedFields]", type ]
+                ready: [ status, conditions, "[type=Ready]", status ]
+                generation: [ status, observedGeneration ]
+                autoRestartCount: [ status, autoRestart, count ]
+                autoRestartConnectorName: [ status, autoRestart, connectorName ]
+                tasksMax: [ status, tasksMax ]
+                topics: [ status, topics ]
+        - groupVersionKind:
+            group: kafka.strimzi.io
+            version: v1beta2
+            kind: KafkaMirrorMaker2
+          metricNamePrefix: strimzi_kafka_mm2
+          metrics:
+            - name: resource_info
+              help: "The current state of a Strimzi Kafka MirrorMaker2 resource."
+              each:
+                type: Info
+                info:
+                  labelsFromPath:
+                    name: [ metadata, name ]
+              labelsFromPath:
+                exported_namespace: [ metadata, namespace ]
+                deprecated: [ status, conditions, "[reason=DeprecatedFields]", type ]
+                ready: [ status, conditions, "[type=Ready]", status ]
+                generation: [ status, observedGeneration ]
+                autoRestartCount: [ status, autoRestart, count ]
+                autoRestartConnectorName: [ status, autoRestart, connectorName ]
```

I believe for MirrorMaker2 these should be `autoRestartStatuses`: https://github.com/strimzi/strimzi-kafka-operator/blob/main/api/src/main/java/io/strimzi/api/kafka/model/mirrormaker2/KafkaMirrorMaker2Status.java#L31

```suggestion
                autoRestartCount: [ status, autoRestartStatuses, count ]
                autoRestartConnectorName: [ status, autoRestartStatuses, connectorName ]
```